### ISSUE TITLE
Should removing support user delete associated posts?

### ISSUE THREAD

**Issue by ethitter** [Automattic/vip-support#55]
When we purge a support user, we remove all of that user's posts. Should we do so, or should we leave those posts in case they were imported or otherwise created at the client's behest?

Related: p2-poqVs-j8o

**Comment by nickdaugherty** [comment]
Deleting posts seems a little dangerous to me, for the reasons you mention. We shouldn't be creating posts on prod sites anyway.

**Comment by ethitter** [comment]
Agreed, it'd be better to leave anything behind. We can assign it to the machine user we use for the JP connection, which should make it obvious where the posts came from.

**Comment by ethitter** [comment]
See also #44.
